scipy/signal/filter_design.py

```diff
@@ -4865,6 +4865,195 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def iircomb(w0, Q, ftype='notch', fs=2.0):
+    """
+    Design IIR notching or peaking digital comb filter.
+
+    A notching comb filter is a band-stop filter with a narrow bandwidth
+    (high quality factor). It rejects a narrow frequency band and
+    leaves the rest of the spectrum little changed.
+
+    A peaking comb filter is a band-pass filter with a narrow bandwidth
+    (high quality factor). It rejects components outside a narrow
+    frequency band.
+
+    Parameters
+    ----------
+    w0 : float
+        Frequency to remove or pass from a signal. If `fs` is specified, this
```

Comment:
I think we need to rework the wording here. Something like "frequency spacing to use". Because the notch filter will filter at 0, w0, 2*w0, ..., but the peak filter will put peaks at 0.5*w0, 1.5*w0. Can you think of a better wording?

Reply:
I'm thinking of including the following under the `ftype` parameter, if that's okay. 

"If 'notch', function returns filter with notches at frequencies `0`, `w0`, `2 * w0`, ... If 'peak', function returns filter with peaks at frequencies `0.5 * w0`, `1.5 * w0`, `2.5 * w0`, ..."

If not, I can modify the `w0` parameter description like you've mentioned.

Reply:
Modifying `ftype` is fine by me

Reply:
Fixed.